FactorioWebInterface/Pages/Admin/CreateAccount.cshtml.cs

```diff
@@ -0,0 +1,106 @@
+﻿using FactorioWebInterface.Data;
+using FactorioWebInterface.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FactorioWebInterface.Pages.Admin
+{
+    [Authorize(Roles = Constants.RootRole)]
```

Comment:
When authorisation is done like this I'm not sure if the return url is set. There might be a way to set that up though, but I'm not sure how. We could just add a role check to when we check if the user is suspended.